docs/ref/models/querysets.txt

```diff
@@ -2582,11 +2582,11 @@ Case-sensitive ends-with.
 
 Example::
 
-    Entry.objects.filter(headline__endswith='cats')
+    Entry.objects.filter(headline__endswith='Cats')
```

Comment:
Still not consistent... any reason for three examples to use "Will" and one use "Cats"?